src/lib/template-result.ts

```diff
@@ -20,6 +20,21 @@ import {reparentNodes} from './dom.js';
 import {TemplateProcessor} from './template-processor.js';
 import {boundAttributeSuffix, lastAttributeNameRegex, marker, nodeMarker} from './template.js';
 
+/**
+ * Our TrustedTypePolicy for HTML which is declared using the html template
+ * tag function.
+ *
+ * That HTML is a developer-authored constant, and is parsed with innerHTML
+ * before any untrusted expressions have been mixed in. Therefor it is
+ * considered safe by construction.
+ */
+const policy = ((): Pick<TrustedTypePolicy, 'createHTML'>|void => {
+  const tt = window.trustedTypes;
+  if (tt !== undefined) {
+    return tt.createPolicy('lit-html', {createHTML: (s) => s});
+  }
+})();
```

Comment:
Consider:

```ts
const policy = trustedTypes && trustedTypes.createPolicy('lit-html', {createHTML: (s) => s});
```

Reply:
I like it. Saves another 14 bytes after gzip

Reply:
Er, 13 bytes.

Adding `window.` adds only one byte to the gzipped output! It's a common enough substring, but that's still impressive!

Reply:
nice :)